### PR TITLE
Add package status filtering to create_aip_replicas command

### PR DESCRIPTION
**Pull request by tw4l**
QA of the new `create_aip_replicas` command by support identified an issue where the command did not work as expected when deleted AIPs were present in the Storage Service database. This PR ensures that packages are properly filtered to only those with status `Package.UPLOADED` to avoid this issue moving forward. I have tested the fix locally with a Storage Service database containing deleted AIPs (including deleted AIP replicas) and verified it is now working as originally intended.

This PR also includes a commit which removes code that explicitly sets a replicated AIP's status to `Package.DELETED` after deletion, since `Package.delete_from_storage()` already does the same.

Connected to https://github.com/archivematica/Issues/issues/1209

### CODE REVIEW

**Comment by amayita** [comment]
I was able to test this in a test setting and it worked as intended for me:
root@am1:~# chown -R archivematica:archivematica /mnt/aipstore* && sudo -u archivematica bash -c " \
set -a -e -x
source /etc/default/archivematica-storage-service || \
source /etc/sysconfig/archivematica-storage-service \
|| (echo 'Environment file not found'; exit 1)
cd /usr/lib/archivematica/storage-service/
/usr/share/archivematica/virtualenvs/archivematica-storage-service/bin/python \
manage.py \
create_aip_replicas --delete
"; 
+ source /etc/default/archivematica-storage-service
++ LANG=en_US.UTF-8
++ LC_ALL=en_US.UTF-8
++ LC_LANG=en_US.UTF-8
++ EMAIL_HOST_PASSWORD=
++ SS_DB_NAME=/var/archivematica/storage-service/storage.db
++ REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
++ DJANGO_ALLOWED_HOSTS='*'
++ SS_DB_PASSWORD=
++ SS_DB_USER=
++ SS_DB_HOST=
++ DJANGO_SETTINGS_MODULE=storage_service.settings.production
++ SS_GUNICORN_BIND=127.0.0.1:8001
++ EMAIL_PORT=25
++ DJANGO_SECRET_KEY=CHANGE_ME_WITH_A_SECRET_KEY
++ EMAIL_HOST_USER=
++ EMAIL_HOST=localhost
+ cd /usr/lib/archivematica/storage-service/
+ /usr/share/archivematica/virtualenvs/archivematica-storage-service/bin/python manage.py create_aip_replicas --delete
**AIPs to replicate: 62**
Deleting existing replicas of AIP fc73b142-5bd9-450c-9284-2132972e5d67
Creating new replicas for AIP fc73b142-5bd9-450c-9284-2132972e5d67
AIP fc73b142-5bd9-450c-9284-2132972e5d67 successfully replicated
[...]
Deleting existing replicas of AIP b891e9f3-2b40-4995-9f3d-43910915dfbe
Creating new replicas for AIP b891e9f3-2b40-4995-9f3d-43910915dfbe
AIP b891e9f3-2b40-4995-9f3d-43910915dfbe successfully replicated
**Replica creation complete. 61 existing replicas deleted. New replicas created for 62 of 62 AIPs in location.**
root@am1:~#

File: storage_service/common/management/commands/create_aip_replicas.py [hunk 1]
```diff
@@ -141,18 +143,15 @@ def _delete_replicas(self, aip_uuid):
         return deleted_count
 
     def _delete_replica(self, replica):
-        """Delete replica from filesystem and database
+        """Delete replica from filesystem and update Package status
+
+        If filesystem deletion is successful, the replica's status will be
+        set to Package.DELETED by `replica.delete_from_storage()`.
```
Comment (ross-spencer) [inline storage_service/common/management/commands/create_aip_replicas.py:149]: Given the context of this second PR I understand this comment. I wonder if this reflects a need to improve the documentation of the function in the storage service code itself... :thinking:

**Review by ross-spencer — APPROVED** [review]
LGTM! And I also performed a play-test too following @amayita's example as this command is helping my development process with another piece.  

How soon is it needed for the client @amayita? We can likely merge this while Tessa is away if it helps move that forward?

File: storage_service/common/management/commands/create_aip_replicas.py (hunk 1 above)
Comment (amayita) [reply storage_service/common/management/commands/create_aip_replicas.py:149]: I am in no hurry, as this version works for me (unless I'm missing something important). 
Please proceed as you see fit.

Comment (ross-spencer) [reply storage_service/common/management/commands/create_aip_replicas.py:149]: @amayita not at all, running this version from the branch seems like a good approach to me. Thanks for the response!

Comment (tw4l) [reply storage_service/common/management/commands/create_aip_replicas.py:149]: @ross-spencer I think in this case the method docstring was clear enough, and the fault was mine for not reading closely.

**Pull request merged**